tests/functional/cylc-cat-log/01-remote.t

```diff
@@ -58,7 +58,7 @@ grep_ok "jumped over the lazy dog" "${TEST_NAME}.out"
 # remote
 TEST_NAME=${TEST_NAME_BASE}-task-status
 cylc cat-log -f s "${WORKFLOW_NAME}//1/a-task" >"${TEST_NAME}.out"
-grep_ok "CYLC_JOB_RUNNER_NAME=at" "${TEST_NAME}.out"
```

Comment:
This would fail if run with anything other than an `at` platform e.g. `_remote_background_indep_tcp`